02_Development/Display/spl06_007.py

```diff
@@ -116,249 +169,69 @@ def __init__(self,
 
     def pressure(self, raw_pressure, raw_temperature):
         scaled_pressure = raw_pressure / self._pressure_scaling_factor
-        scaled_temperature = raw_temperature / self._temperature_scaling_factor
+        scaled_temperature = (raw_temperature
+                              / self._temperature_scaling_factor)
 
-        compensated_pressure = (                                       
+        compensated_pressure = (
             self._c00
             + scaled_pressure*(self._c10
                                + scaled_pressure*(self._c20
```

Comment:
this spacing is weird

Reply:
I found it helpful while checking against the data sheet to make sure that this equation is correct.  This way it's very easy to see the equation structure and which term everything is a part of.  Granted it is very strange without context.